### PR TITLE
Added `Kubescape` project

### LINKED ISSUE #158: Adding OSS project to `Devops/contribute` folder
### Which field of community (DevOps/Opensource/Web/Web3/Android/Devrel) you want to add, drop the links?

I want to add `kubescape` project 
link here:  https://github.com/kubescape/kubescape

### For adding Job Posting, add links here.

_No response_

### For adding OpenSource Projects, add links here.

_No response_

### PR DESCRIPTION
**Pull request by Pranav-Shinde11**
closes #158

### CODE REVIEW

**Comment by Pranav-Shinde11** [comment]
Hi @kaiwalyakoparkar
I made PR  can you review it?

**Review by siddhant-khisty — APPROVED** [review]
LGTM

**Review by SaptarshiSarkar12 — APPROVED** [review]
The code looks good to merge 👍. The changes can be merged.

File: DevOps/Contribute/Readme.md [hunk 1]
```diff
@@ -26,3 +26,4 @@
 - [rclone](https://github.com/rclone)
 - [Stemformatics](https://github.com/wellslab/s4m-api)
 - [Wireshark](https://gitlab.com/wireshark/wireshark)
+- [Kubescape](https://github.com/kubescape/kubescape)
```
Comment (kaiwalyakoparkar) [inline DevOps/Contribute/Readme.md:29]: Please add name according to the alphabetical order

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: DevOps/Contribute/Readme.md (hunk 1 above)
Comment (Pranav-Shinde11) [reply DevOps/Contribute/Readme.md]: Name added according to the alphabetical order.
Please review.

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm, thank you so much for contributing

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**